hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/snapshot/TestOMSnapshotDeleteResponse.java

```diff
@@ -40,6 +44,9 @@
 import java.io.File;
 import java.util.UUID;
 
+import static org.apache.hadoop.ozone.om.helpers.SnapshotInfo.SnapshotStatus.SNAPSHOT_ACTIVE;
+import static org.apache.hadoop.ozone.om.helpers.SnapshotInfo.SnapshotStatus.SNAPSHOT_DELETED;
+
 
 /**
  * This class tests OMSnapshotDeleteResponse.
```

Comment:
We can remove the `TODO` line below since this PR completes it.

Reply:
Good point. Done